dev/run-tests-jenkins.py

```diff
@@ -183,6 +183,7 @@ def main():
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop2.7"
     if "test-hadoop3.2" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop3.2"
+        os.environ["JAVA_HOME"] = "/usr/java/jdk-11.0.1"
```

Comment:
If the job fails at `unidoc` stage, please try to comment out it in `run-tests.py`.
```
-    build_spark_unidoc_sbt(hadoop_version)
+    # build_spark_unidoc_sbt(hadoop_version)
```

Reply:
Thank you @dongjoon-hyun

Reply:
Let's perhaps look separately for `test-java11` as in https://github.com/apache/spark/pull/25423 so we can separate these concerns?

So let's see this script only runs tests, so yes it should set JAVA_HOME this way when testing Java 11. I think `dev/run-tests` has to change too, to set `-Djava.version=11` to Maven and SBT when testing Java 11 too. Can we try that here?

Reply:
i am fine w/keeping this in here for the purposes of testing this PR, but i'd really like to keep the build/test logic for java11 in one PR...  it's already confusing enough as it is.  :)

Reply:
Thank you @shaneknapp keeping this in here just for the purposes of testing this PR. I will remove this once  #25423 merged.